packages/system/src/Box/stories.mdx

```diff
@@ -0,0 +1,100 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { Box } from './Box';
+
+<Meta title="MDX/Box Component Doc" />
+
+# Box-Component
+
+<br />
+
+## Description
+
+`Box` is a mutli-purpose component and the most abstract component on top of which all other Marigold UI components are built.
+The `Box` allows us to apply styling via a dedicated prop (`css`) while respecting the rules and constraints of our design system.
+Instead of exposing the underlying tools that will create CSS, the `Box` component wraps them and exposes a more React-like API for applying styling to a component.
+
+<br />
```

Comment:
You are using a lot of <br>s. This shouldn't be necessary

Reply:
A lot of <br /> or a lot of the letter s ?